contributors/design-proposals/customresources-subresources.md

```diff
@@ -75,13 +75,18 @@ type CustomResourceSubResourceStatus struct {
 type CustomResourceSubResourceScale struct {
     // required, e.g. “.spec.replicas”. Must be under `.spec`.
     // Only JSON paths without the array notation are allowed.
-    SpecReplicasPath string `json:"specReplicasPath,omitempty"`
+    SpecReplicasPath string `json:"specReplicasPath"`
     // optional, e.g. “.status.replicas”. Must be under `.status`.
     // Only JSON paths without the array notation are allowed.
     StatusReplicasPath string `json:"statusReplicasPath,omitempty"`
     // optional, e.g. “.spec.labelSelector”. Must be under `.spec`.
     // Only JSON paths without the array notation are allowed.
     LabelSelectorPath string `json:"labelSelectorPath,omitempty"`
+    // ScaleGroupVersion denotes the GroupVersion of the Scale
+    // object sent as the payload for /scale. It allows transition
+    // to future versions easily.
+    // Today only autoscaling/v1 is allowed.
+    ScaleGroupVersion schema.GroupVersion `json:"groupVersion"`
```

Comment:
this is a logical type, but don't actually embed the type here

Reply:
The external type will have a string, the internal the logical one.

Reply:
> The external type will have a string, the internal the logical one.

Ok.